texastribune/newsapps-styleguide-2.0
# Updates

#### What's the plan?
Once these changes are team approved, we'll need to copy the new `/package` folder and replace it in the newsapps-styles repo, publish a new version and update newsapps kits.

#### What's this PR do?
+ Converts Style Guide to new-lessons from app kit, restructures HTML templates to match new kit format
+ Adds `/styles/package` to test styles that will be updated and transferred to the newsapps-styles package
+ Wholesale removes _base.scss file and all styles applied directly to HTML elements
+ Removes Bourbon/Neat dependency, adds a replacement `@mixin media` for simple media queries and substitutes for neat `@mixin media` in table styles, typography file
+ Removes components folder and styles

#### Why are we doing this? How does it help us?
+ Cleans up newsapps-styles package, so that we don't have to override bad code and makes it more flexible for designers/developers to adapt to their needs.

#### How should this be manually tested?
+ Currently tested by applying styles within the guide from the /package folder, so you can see them in action w/ `npm run serve`

#### Have automated tests been added?
NA

#### Next steps?
Update newsapps-styles repo/release new version/update other dependent kits.
Discuss next additions as a team.

#### Has the relevant README been updated?
Yes
